### DISCUSSION TITLE
Sequent Super Watchdog issue

### DISCUSSION THREAD

**Discussion by chadhobson** [tagyoureit/nodejs-poolController#823] [Nixie]
Just added the Sequent Super Watchdog to my controller but having an issue...

Alexandru @ Sequent says I need to set the MEGA-BAS Watchdog to match the Watchdog but when I try to look at the Watchdog under MEGA-BAS I see this (an empty box)...

![Screenshot 2023-07-10 at 6 37 12 PM](https://github.com/tagyoureit/nodejs-poolController/assets/1839161/b316790c-9474-4437-9059-ba3aae84686f)

Any idea what I'm missing? No errors in the browser console, FYI.

**Comment by rstrouse** [comment]
The MEGA-BAS and Super Watchdog are two different devices.  I am not sure what Alex is talking about though, the watchdog should only be enabled on one device and the Super Watchdog actually contains a battery backup.  We decided against implementation on the BAS since it was a bit of a booger in the firmware and removed the controls from the output.  It should be disabled on the BAS as there is nothing resource-wise being controlled there.

**Reply by chadhobson** [reply]
His exact comment...

"If you need the watchdog function (cycle the RPI power when hung) you need to set the Building Automation watchdog function with the same period as for the superwatchdog"
